mtp_api/apps/prison/models.py

```diff
@@ -23,9 +23,12 @@ class PrisonerLocation(TimeStampedModel):
     prison = models.ForeignKey(Prison)
 
     class Meta:
-        index_together = [
-            ["prisoner_number", "prisoner_dob"],
-        ]
+        index_together = (
+            ('prisoner_number', 'prisoner_dob'),
+        )
+        permissions = (
+            ('identify_prisoner', 'Can identify if a prisoner exists'),
```

Comment:
actually, action-based permissions aren't needed as there's only one method and the api user is shared for all calls from the send-money app